config.toml

```diff
@@ -28,11 +28,15 @@ ignoreFiles = []
 ###############################################################################
 [menu]
   [[menu.main]]
-    name = "Kubeflow Summit 2023 "
+    name = "Google Summer Of Code 2024 "
     weight = -1000
     pre = "<i class='fas fa-calendar pr-2' style='color: #FFC107'></i>"
-    post = "<br><span class='badge badge-warning'>Oct 6th</span> <span class='badge badge-warning'>Irving, Texas</span> <span class='badge badge-warning'>Virtual</span>"
-    url = "/events/kubeflow-summit-2023/"
+    url = "/events/gsoc-2024/"
```

Comment:
Personally, I think the "highlighted events" should be for actual in-person events that typical end-users can attend, and it's unlikely that GSOC will be relevant to most end-users of Kubeflow.

So let's just comment out the existing "Kubeflow Summit 2023" for now, so we can use it as a template for the next event.

If you are able, we could also make a page for Kubecon EU 2024, which Kubeflow is going to have a presence at, and make that the "highlighted" event.  (FYI, it's on `19 – 22 MARCH, 2024`, but I need to confirm who has the info about what we are doing there before we list it as a page).

Reply:
Agree.

So for now I added Kubecon but going to the Kubecon page, once we have the agenda or presenters then we will add the landing page. This can be a nice placement until then?